packages/application-config/client.d.ts

```diff
@@ -1,5 +1,13 @@
 type CSSModuleClasses = { readonly [key: string]: string };
 
+declare namespace NodeJS {
+  // Used for global variable replacements.
+  interface ProcessEnv {
+    readonly DEBUG: 'true' | 'false';
+    readonly NODE_ENV: 'development' | 'production' | 'test';
+  }
+}
```

Comment:
Slightly unrelated but noticed this in the CRA setup: https://github.com/facebook/create-react-app/blob/f34d88e30c7d8be7181f728d1abc4fd8d5cd07d3/packages/react-scripts/lib/react-app.d.ts#L5-L10